web/js/jquery.websocket.js

```diff
@@ -35,7 +35,12 @@
                     .bind('message', function(e) {
                         var m = JSON.parse(e.originalEvent.data);
                         var h = settings.events[m.type];
-                        if (h) h.call(this, m);
```

Comment:
This is an external library that I would rather be able to upgrade merely by copying in the updated library. Can this error be moved into code we wrote?